src/core/tests/visitors/op/istft.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/istft.hpp"
+
+#include <gtest/gtest.h>
+
+#include "openvino/op/constant.hpp"
+#include "openvino/op/parameter.hpp"
+#include "visitors/visitors.hpp"
+
+namespace ov {
+namespace test {
```

Comment:
```suggestion
namespace ov::test {

using op::v0::Paramter, op::v0::Constant;
```
Like in type prop the create node will be shorter in tests

Reply:
Done